utils/cq/requirements.txt

```diff
@@ -9,3 +9,5 @@ yamllint==1.35.1
 codespell==2.2.6
 # Used by ci/jira_query.py which pip installs it standalone.
 jira
+# Used by ftest/tags.py
+yaml
```

Comment:
Even without the above two changes, my PR seems to work... I just added the 73-ftest.sh changes alone on my PR...

Reply:
Oh, you reverted it 👍

Reply:
Thanks @rpadma2. It turns out `yaml` is the wrong name. But `yamllint` pulls in `pyyaml`, which is what your env was missing anyway

Reply:
So the requirements.txt was already good :)